packages/select/src/components/query-list/queryList.tsx

```diff
@@ -422,7 +431,9 @@ export class QueryList<T> extends AbstractComponent2<QueryListProps<T>, IQueryLi
                 return this.itemsParentRef.children.item(index) as HTMLElement;
```

Comment:
Strictly speaking, we could also wire up a ref to `createNewItemRenderer` so that we can easily look it up, but I think even without doing so `0` and `this.state.filteredItems.length` should be good enough as indices.